### DISCUSSION TITLE
Where to install IMAGICK ?

### DISCUSSION THREAD

**Discussion by gestyle** [ImageMagick/ImageMagick#7591] [Help]
### ImageMagick version

1.7.9

### Operating system, version and so on

Linux 4.4.400-icpu-099 x86_64

### Description

Hello,

I’m struggling with the IMAGICK module, as I’m not sure where it should be installed within my project on a remote server. Here is the thread to the already installed IMAGEMAGICK Engine 1.7.9. :

/project_1/wp-content/plugins/imagemagick-engine

Where now am I supposed to install IMAGICK to ? 
…in 
/project_1/wp-content/plugins/imagemagick-engine ?
/project_1/wp-content/plugins ?
/project_1/wp-content ?

Any advice would be much appreciated.

:)

**Comment by dlemstra** [comment]
We don't maintain that. I would advise you to ask that question here: https://github.com/Imagick/imagick.

**Comment by gestyle** [comment]
Merci dlemstra, I’ll try to figure it out there :)
